camera_to_hls/lib/membrane_demo_camera_to_hls/pipeline.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.Demo.CameraToHls.Pipeline do
+  use Membrane.Pipeline
+
+  @impl true
+  def handle_init(_) do
+    children = [
+      # Captures video from the camera (raw video, depending on camera/os)
+      source: Membrane.CameraCapture,
+      # Converts pixel format to I420 (this is still a raw video, but different format)
+      converter: %Membrane.FFmpeg.SWScale.PixelFormatConverter{format: :I420},
+      # Takes raw I420 frames and converts to H264
+      # baseline profile tells encoder not to generate
+      # B-frames because browsers can't handle them
+      encoder: %Membrane.H264.FFmpeg.Encoder{profile: :baseline},
+      # Creates caps, generates metadata and adds timestamps
+      # There we need to specify framerate, based on it we
+      # generate timestamps
+      # Further we won't be able to control framerate
+      video_nal_parser: %Membrane.H264.FFmpeg.Parser{
+        framerate: {30, 1},
+        alignment: :au,
+        attach_nalus?: true
+      },
+      # Generates caps for mp4 by changing H264
+      # It is the same H264, only difference is the way of division
```

Comment:
Changing H264? You're not being precise. H264 is converted for Annex B form to length prefixed form. This is called delimitation, not `the way of division`. Also, this is quite important, this elements payloads the H264 so that it can be injected into MP4